docs/docs/documentation/basics.md

```diff
@@ -0,0 +1,303 @@
+```python
+pipeline.predict(subject='Hi!', body='Hi, hope you are well..')
+```
+
+### Explain
+Another thing we can do with a trained pipeline is to use the [explain](../api/biome/text/pipeline.md#explain) method to get the attribution of each token by means of [integrated gradients](https://arxiv.org/abs/1703.01365) for those heads that support it:
+```python
+pipeline.explain(text='Good movie indeed!')
+```
+
+### Serve
+Pipeline use the awesome [FastAPI](https://fastapi.tiangolo.com/) library to give you simple REST endpoint with the [serve](../api/biome/text/pipeline.md#serve) method, which provides methods aligned with your input features (e.g., a method accepting `subject` and `body` parameters):
+
+```python
+pipeline.serve(port=9090)
+```
+
+### Explore
+In order to support users with fine-grained error analysis and empower them to improve their models with informed decisions, the [explore](../api/biome/text/pipeline.md#explore) method launches a UI (inside your notebook if you are using Jupyter).
+
+```python
+pipeline.explore(
+    train_ds,
+    explain=True
+)
+```
+
+This search-based UI can help:
+
+ - to quickly identify difficult examples in the training, validation, test or any other data set.
+ - to inform users about corner cases and which labels need more examples.
+ - to find the values for confidence thresholds to use the model in production.
+ - to help domain experts understand the strengths and limitations of the model.
+ - in summary, get a sense of the model interacting with data.
+
+:::tip
+For the UI to work you need a running [Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/current/install-elasticsearch.html) instance. We recommend installing [Elasticsearch with docker](https://www.elastic.co/guide/en/elasticsearch/reference/7.7/docker.html#docker-cli-run-dev-mode).
+:::
+
+### Training and transfer learning
+
+After loading a pipeline, you can keep on training it for the same task with new data:
+```python
+training_results = pipeline.train(
+    output="path_to_store_further_training",
+    training=new_training_ds
+)
+```
+:::tip
+For further training, you can use the [extend_vocab](../api/biome/text/pipeline.md#pipeline) parameter to extend the vocabulary with the new data sets.
+:::
+
+Another thing you can do is to use this pre-trained pipeline for related task, for example another classifier with different labels but a similar domain:
+```python{3-6}
+from biome.text.modules.heads import TextClassification
+
+pipeline.set_head(
+    TextClassification,
+    labels = ["horror_movie", "comedy", "drama", "social"]
+)
+
+training_results = pipeline.train(
+    output="path_to_fine_tuned",
+    training=categories_training_ds
+)
+```
+Here we just use the [set_head](../api/biome/text/pipeline.md#set-head) method to set a new tasks head which classifies film categories instead of review sentiment.
```

Comment:
```suggestion
Here we just use the [Pipeline.set_head()](../api/biome/text/pipeline.md#set-head) method to set a new task head which classifies film categories instead of review sentiment.
```